django-markdown/django_markdown/README.md

```diff
@@ -0,0 +1,3 @@
+# How to Render Markdown in a Django Application
```

Comment:
Can you move the README file one level up, please? Alternatively, I'm wondering if we need the extra subfolder at all. Maybe that would be a better option, actually:

```
materials/
└── django_markdown/
    ├── django_markdown/
    ├── django_markdown_app/
    ├── manage.py
    ├── README.md
    └── requirements.txt
```

Reply:
I added that top level `django-markdown` folder based on the instructions above: 'New files should go into a top-level subfolder, named after the article slug. For example: my-awesome-article'. 

But I think since I used an underscore in the project name in the tutorial, `django_markdown`, I will just move the readme file up to the the top level alongside the Django project folder to be consistent. It looks like some of the other Django projects in here are done this way.